web/exercise-05/details.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pokemons Info</title>
+    <style>
+        body {
+            font-family: sans-serif;
+        }
+
+        p {
+            padding-bottom: 0.5rem;
+            border-bottom: 2px solid #ccc;
+        }
+
+        p:nth-of-type(4), p:last-of-type{
+            display: inline-block;
+            border: none;
+        }
+
+        #show_more {
+            background-color: antiquewhite;
+            padding: 0.5rem;
+            cursor: pointer;
+        }
+
+        @media only screen and (max-width: 600px) {
+            body {
+                font-size: 70%;
+            }
+        }
+    </style>
+</head>
+<body>
+    <h1>Info</h1>
+    <div class="details">
+        <p id="name"></p>
+        <p id="stats"></p>
+        <p id="forms"></p>
+        <p id="moves"></p>
+        <p id="show_more" onclick="show_more_moves()">Show More Moves</p>
+    </div>
+
+    <script>
+        const API_URL = 'https://pokeapi.co/api/v2/pokemon'
+        const id = `${window.location.href}`.slice(-1)
+        let pokemon_moves_arr = []
+        console.log(id)
+        
+
+        const fetchMoreDetails= async (id) => {
+            try {
+                const response = await fetch(`${API_URL}/${id}`)
+                const pokemon_details = await response.json()
+                return pokemon_details
+            }
+            catch(err) {
+                return err.message
+            }
+        }
+
+        fetchMoreDetails(id).then(pokemon_details => {
+                let {species, stats, moves, forms} = pokemon_details
+
+                let species_name = species.name
+                let pokemon_stats = stats.map(s => s.stat.name)
+                let pokemon_moves = moves.map(m => pokemon_moves_arr.push(m.move.name))
+                let pokemon_forms = forms.map(f => f.name)
+
+                document.getElementById('name').innerHTML = 'Name: ' +species_name
+                document.getElementById('stats').innerHTML = 'Stats: ' + pokemon_stats
+                document.getElementById('forms').innerHTML = 'Form: ' + pokemon_forms
+                document.getElementById('moves').innerHTML = 'Moves: ' + pokemon_moves_arr.slice(0, 5)
+            }
+        )
+
+        const show_more_moves = () => {
+            const show_more_btn = document.getElementById('show_more')
+            document.getElementById('moves').innerHTML = 'Moves: ' + pokemon_moves_arr.slice(5, pokemon_moves_arr.length)
+            show_more_btn.style.display = 'none'
```

Comment:
Instead of hiding the button, we could change it to show less moves and toggle the display the moves that way